Dockerfile

```diff
@@ -21,7 +20,8 @@ RUN buildDeps=' \
 	&& apk update \
 	&& apk add $buildDeps \
 	&& cd /go/src/github.com/docker/opensource \
-	&& go get -d -v github.com/docker/opensource/maintainercollector \
+	&& go get github.com/BurntSushi/toml \
+	&& go get github.com/sirupsen/logrus \
```

Comment:
And these probably would no longer be needed if we make a module of this repo

Not urgent, so let's get this one going